.github/workflows/check_release_due.yml

```diff
@@ -0,0 +1,39 @@
+name: Check if release is due
+
+on:
+  schedule:
+    # Runs every Monday at 06:15
+    - cron: "15 6 * * 0"
+
+jobs:
+  check-release-due:
+    # We want to create a new release every 6 weeks. I can't work out a way to
+    # do this with the schedule step's cron syntax. Instead, trigger this job
+    # every Monday, then check if the current date is a multiple of 6 weeks
+    # from the very first release date.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Date
+        id: check-date
+        run: |
+          INITIAL_RELEASE_DATE="$(date --date="220530" +%s)"
+          CURRENT_DATE="$(date --date="$(date +'%y%m%d')" +%s)"
+          DAYS_DIFF=$(( ("${INITIAL_RELEASE_DATE}" - "${CURRENT_DATE}")/(60*60*24) ))
+          if [ $(("${DAYS_DIFF}" % 42)) -eq 0 ]; then
+            echo ::set-output name=release_due::true
+          else
+            echo "Release is not due"
+          fi
+      - name: Checkout
+        uses: actions/checkout@v3
+        if: ${{ steps.check-date.outputs.release_due == 'true' }}
+      - name: Create Issue
+        id: create-issue
+        if: ${{ steps.check-date.outputs.release_due == 'true' }}
+        run: >
+          gh issue create \
```

Comment:
Just a note for the conversation we're having elsewhere:

We could/should ff merge for the develop_dependencies too also separately to this we should find a way torebase develop_dependencies and resign the relevant commits